week02_classification/seminar.ipynb

```diff
@@ -637,7 +637,7 @@
     "        data_drop_one_token.loc[drop_i, col_name] = ' '.join(UNK if i == drop_i else tok\n",
     "                                                   for i, tok in enumerate(sample_col_tokens)) \n",
     "\n",
-    "    *predictions_drop_one_token, baseline_pred = model.predict(make_batch(data_drop_one_token, device=device)).detach().cpu()\n",
+    "    *predictions_drop_one_token, baseline_pred = model(make_batch(data_drop_one_token, device=device)).detach().cpu()\n",
```

Comment:
this is a correct change, though nobody complained so far. 
Looks like a leftover from Keras conversion